ush/create_aqm_rc_file.py

```diff
@@ -136,17 +134,29 @@ def create_aqm_rc_file(cdate, run_dir, init_concentrations):
             suffix=".yaml") as tmpfile:
         tmpfile.write(settings_str)
         tmpfile.seek(0)
-        set_template(
-            [
-                "-q",
-                "-c",
-                tmpfile.name,
+        cmd = " ".join(["uw template render",
                 "-i",
                 AQM_RC_TMPL_FP,
                 "-o",
                 aqm_rc_fp,
+                "-v",
+                "--values-file",
+                tmpfile.name,
             ]
         )
+        indent = "  "
+        output = ""
+        try:
+            output = check_output(cmd, encoding="utf=8", shell=True,
+                    stderr=STDOUT, text=True)
```

Comment:
This is similarly called in several python scripts that create various config files. You'll see this pattern repeated below.

This runs a subprocess and exits if there are failures from that.

Reply:
I understand that this looks a bit more clunky in this iteration. The UW Team is currently preparing a Python API that will be able to work just as you describe, and will be available in our next released package (hopefully by January). We will open a PR as soon as we can to update this (and others like it) to a call to that API. It will look something like:

```
from uwtools.api import template
...
template.render(
    values = settings,
    input_file = AQM_RC_TMPL_FP,
    output_file = aqm_rc_fp,
    )
```

This won't require the temporary file or the subprocess to work, but will use the settings dict directly.

Our approach here is deliberate so that we can expose and protect certain behaviors in a well-defined API interface to ensure that we do not change those functions when we update versions. While end users are, of course, able to import any portion of the uwtools package anywhere in their code base, we highly recommend against the practice. If, instead, it is done through our defined API functions, we are guaranteeing that functionality will not be modified in a breaking way in subsequent releases of our package without sufficient notification. The API will essentially be a contract between uwtools and its end users about the functionality it will continue to maintain.

Reply:
Good to hear that this is an intermediate step, I look forward to the simpler interface in the future!